.eslintrc.js

```diff
@@ -0,0 +1,137 @@
+module.exports = {
```

Comment:
You can use our supercool(tm) company wide eslint available here: https://github.com/nqminds/nqm-utils/tree/master/packages/eslint-config

Reply:
Thanks.

Reply:
Looks very neat.